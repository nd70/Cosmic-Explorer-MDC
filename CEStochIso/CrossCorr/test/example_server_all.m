%
% Run all jobs in the examnple job file to vaildate
% post-processing
%

stochastic_paths;
delete('example_server_all.out');
[success, msg, msgid] = rmdir('cachefiles_server', 's');
mkdir('cachefiles_server');
diary('example_server_all.out');
stochastic('example_server_all.txt', 'example_jobs.txt', 1);
stochastic('example_server_all.txt', 'example_jobs.txt', 2);
stochastic('example_server_all.txt', 'example_jobs.txt', 3);
diary off;
