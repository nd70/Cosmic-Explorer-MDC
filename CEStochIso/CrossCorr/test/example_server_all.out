segment=3, trial =1, start sec = 1126626075, CC stat = -1.610751e-04, theor sigma = 4.556855e-04
segment=4, trial =1, start sec = 1126626105, CC stat = -5.317729e-04, theor sigma = 4.632246e-04
segment=5, trial =1, start sec = 1126626135, CC stat = 2.933996e-04, theor sigma = 4.670951e-04
segment=6, trial =1, start sec = 1126626165, CC stat = 3.564687e-04, theor sigma = 4.608404e-04
segment=7, trial =1, start sec = 1126626195, CC stat = 2.825762e-04, theor sigma = 4.611440e-04
segment=8, trial =1, start sec = 1126626225, CC stat = -3.154353e-04, theor sigma = 4.581993e-04
segment=9, trial =1, start sec = 1126626255, CC stat = 2.459836e-04, theor sigma = 4.594404e-04
segment=10, trial =1, start sec = 1126626285, CC stat = 7.509449e-04, theor sigma = 4.547512e-04
segment=11, trial =1, start sec = 1126626315, CC stat = 9.446346e-06, theor sigma = 4.539911e-04
segment=12, trial =1, start sec = 1126626345, CC stat = -4.455225e-04, theor sigma = 4.421551e-04
segment=13, trial =1, start sec = 1126626375, CC stat = 3.781912e-04, theor sigma = 4.383439e-04
segment=14, trial =1, start sec = 1126626405, CC stat = 4.334577e-04, theor sigma = 4.400128e-04
segment=15, trial =1, start sec = 1126626435, CC stat = -5.706637e-04, theor sigma = 4.381494e-04
segment=16, trial =1, start sec = 1126626465, CC stat = -2.164340e-04, theor sigma = 4.370426e-04
segment=17, trial =1, start sec = 1126626495, CC stat = -8.395722e-04, theor sigma = 4.330195e-04
segment=18, trial =1, start sec = 1126626525, CC stat = -1.095945e-03, theor sigma = 4.265921e-04
segment=19, trial =1, start sec = 1126626555, CC stat = 4.596465e-04, theor sigma = 4.290315e-04
segment=20, trial =1, start sec = 1126626585, CC stat = -6.365942e-05, theor sigma = 4.439318e-04
segment=21, trial =1, start sec = 1126626615, CC stat = 4.523975e-04, theor sigma = 4.494876e-04
segment=22, trial =1, start sec = 1126626645, CC stat = 1.622646e-04, theor sigma = 4.452459e-04
segment=23, trial =1, start sec = 1126626675, CC stat = -2.868838e-04, theor sigma = 4.333874e-04
segment=24, trial =1, start sec = 1126626705, CC stat = 3.698967e-04, theor sigma = 4.352248e-04
segment=25, trial =1, start sec = 1126626735, CC stat = 5.270730e-04, theor sigma = 4.448423e-04
segment=26, trial =1, start sec = 1126626765, CC stat = 5.086034e-04, theor sigma = 4.650624e-04
segment=27, trial =1, start sec = 1126626795, CC stat = -1.988094e-04, theor sigma = 4.585238e-04
segment=28, trial =1, start sec = 1126626825, CC stat = 2.549785e-04, theor sigma = 4.492477e-04
segment=29, trial =1, start sec = 1126626855, CC stat = 5.085108e-04, theor sigma = 4.494080e-04
segment=30, trial =1, start sec = 1126626885, CC stat = 3.134164e-04, theor sigma = 4.586114e-04
segment=31, trial =1, start sec = 1126626915, CC stat = -7.783852e-06, theor sigma = 4.631874e-04
segment=32, trial =1, start sec = 1126626945, CC stat = -3.925304e-04, theor sigma = 4.598011e-04
segment=33, trial =1, start sec = 1126626975, CC stat = -4.352466e-04, theor sigma = 4.790235e-04
segment=34, trial =1, start sec = 1126627005, CC stat = 3.984703e-04, theor sigma = 4.748823e-04
segment=35, trial =1, start sec = 1126627035, CC stat = 9.681774e-05, theor sigma = 4.552185e-04
segment=36, trial =1, start sec = 1126627065, CC stat = 4.286636e-04, theor sigma = 4.500477e-04
segment=37, trial =1, start sec = 1126627095, CC stat = 1.291056e-04, theor sigma = 4.710300e-04
segment=38, trial =1, start sec = 1126627125, CC stat = 6.644900e-04, theor sigma = 4.750279e-04
segment=39, trial =1, start sec = 1126627155, CC stat = 3.511126e-04, theor sigma = 4.502484e-04
segment=3, trial =1, start sec = 1126627369, CC stat = 3.070804e-04, theor sigma = 4.385573e-04
segment=4, trial =1, start sec = 1126627399, CC stat = -4.721304e-05, theor sigma = 4.331303e-04
segment=5, trial =1, start sec = 1126627429, CC stat = 3.808077e-04, theor sigma = 4.276659e-04
segment=6, trial =1, start sec = 1126627459, CC stat = 6.348086e-04, theor sigma = 4.308190e-04
segment=7, trial =1, start sec = 1126627489, CC stat = -2.095123e-04, theor sigma = 4.289766e-04
segment=8, trial =1, start sec = 1126627519, CC stat = 3.236710e-04, theor sigma = 4.204094e-04
segment=9, trial =1, start sec = 1126627549, CC stat = 3.416989e-04, theor sigma = 4.170852e-04
segment=10, trial =1, start sec = 1126627579, CC stat = 2.491606e-04, theor sigma = 4.275095e-04
segment=11, trial =1, start sec = 1126627609, CC stat = 1.435865e-04, theor sigma = 4.425026e-04
segment=12, trial =1, start sec = 1126627639, CC stat = -4.610263e-04, theor sigma = 4.603682e-04
segment=13, trial =1, start sec = 1126627669, CC stat = -7.082224e-04, theor sigma = 4.507563e-04
segment=14, trial =1, start sec = 1126627699, CC stat = 1.889514e-04, theor sigma = 4.548161e-04
segment=15, trial =1, start sec = 1126627729, CC stat = -1.111019e-04, theor sigma = 4.899160e-04
segment=16, trial =1, start sec = 1126627759, CC stat = 1.848009e-04, theor sigma = 5.036686e-04
segment=17, trial =1, start sec = 1126627789, CC stat = -6.454592e-04, theor sigma = 4.764881e-04
segment=18, trial =1, start sec = 1126627819, CC stat = 1.081851e-03, theor sigma = 4.761411e-04
segment=19, trial =1, start sec = 1126627849, CC stat = -4.415706e-04, theor sigma = 4.827310e-04
segment=20, trial =1, start sec = 1126627879, CC stat = -5.005905e-04, theor sigma = 4.671271e-04
segment=21, trial =1, start sec = 1126627909, CC stat = 4.480586e-04, theor sigma = 4.535013e-04
segment=22, trial =1, start sec = 1126627939, CC stat = 2.599749e-04, theor sigma = 4.543806e-04
segment=23, trial =1, start sec = 1126627969, CC stat = -4.062091e-04, theor sigma = 4.388201e-04
segment=3, trial =1, start sec = 1126643603, CC stat = 5.597523e-04, theor sigma = 4.145244e-04
segment=4, trial =1, start sec = 1126643633, CC stat = 4.548432e-05, theor sigma = 4.289550e-04
segment=5, trial =1, start sec = 1126643663, CC stat = -2.131570e-04, theor sigma = 4.390990e-04
segment=6, trial =1, start sec = 1126643693, CC stat = -9.293389e-04, theor sigma = 4.239447e-04
segment=7, trial =1, start sec = 1126643723, CC stat = 1.020191e-03, theor sigma = 4.138207e-04
segment=8, trial =1, start sec = 1126643753, CC stat = 6.817726e-04, theor sigma = 4.117593e-04
segment=9, trial =1, start sec = 1126643783, CC stat = -7.305027e-05, theor sigma = 4.134669e-04
segment=10, trial =1, start sec = 1126643813, CC stat = 4.326493e-04, theor sigma = 4.137791e-04
segment=11, trial =1, start sec = 1126643843, CC stat = -1.080139e-04, theor sigma = 4.066223e-04
segment=12, trial =1, start sec = 1126643873, CC stat = -5.622439e-04, theor sigma = 4.073220e-04
segment=13, trial =1, start sec = 1126643903, CC stat = -6.418782e-05, theor sigma = 4.183537e-04
segment=14, trial =1, start sec = 1126643933, CC stat = -2.308930e-04, theor sigma = 4.203017e-04
segment=15, trial =1, start sec = 1126643963, CC stat = 6.916928e-04, theor sigma = 4.116834e-04
segment=16, trial =1, start sec = 1126643993, CC stat = -2.682871e-04, theor sigma = 4.062399e-04
segment=17, trial =1, start sec = 1126644023, CC stat = 2.857918e-05, theor sigma = 4.121159e-04
segment=18, trial =1, start sec = 1126644053, CC stat = 1.602234e-04, theor sigma = 4.221714e-04
segment=19, trial =1, start sec = 1126644083, CC stat = -2.335432e-04, theor sigma = 4.202133e-04
segment=20, trial =1, start sec = 1126644113, CC stat = -3.518909e-04, theor sigma = 4.041529e-04
segment=21, trial =1, start sec = 1126644143, CC stat = 3.317762e-04, theor sigma = 4.017700e-04
segment=22, trial =1, start sec = 1126644173, CC stat = 6.285872e-04, theor sigma = 4.131176e-04
segment=23, trial =1, start sec = 1126644203, CC stat = -2.250653e-04, theor sigma = 4.192425e-04
segment=24, trial =1, start sec = 1126644233, CC stat = 2.721413e-04, theor sigma = 4.088694e-04
segment=25, trial =1, start sec = 1126644263, CC stat = -4.904645e-04, theor sigma = 4.142993e-04
segment=26, trial =1, start sec = 1126644293, CC stat = -1.240604e-04, theor sigma = 4.207051e-04
segment=27, trial =1, start sec = 1126644323, CC stat = -5.510188e-04, theor sigma = 4.122189e-04
segment=28, trial =1, start sec = 1126644353, CC stat = 3.428318e-04, theor sigma = 4.057525e-04
segment=29, trial =1, start sec = 1126644383, CC stat = -4.872291e-05, theor sigma = 4.071959e-04
segment=30, trial =1, start sec = 1126644413, CC stat = 5.000835e-04, theor sigma = 4.162706e-04
segment=31, trial =1, start sec = 1126644443, CC stat = 4.545110e-04, theor sigma = 4.126629e-04
segment=32, trial =1, start sec = 1126644473, CC stat = 2.621791e-04, theor sigma = 3.989438e-04
segment=33, trial =1, start sec = 1126644503, CC stat = 2.616268e-04, theor sigma = 4.028635e-04
segment=34, trial =1, start sec = 1126644533, CC stat = -1.018740e-04, theor sigma = 4.100918e-04
segment=35, trial =1, start sec = 1126644563, CC stat = 2.936213e-04, theor sigma = 4.237250e-04
segment=36, trial =1, start sec = 1126644593, CC stat = 4.276039e-04, theor sigma = 4.143031e-04
segment=37, trial =1, start sec = 1126644623, CC stat = -4.210991e-04, theor sigma = 4.114171e-04
segment=38, trial =1, start sec = 1126644653, CC stat = -1.480465e-05, theor sigma = 4.061845e-04
segment=39, trial =1, start sec = 1126644683, CC stat = 1.791653e-04, theor sigma = 4.177789e-04
segment=40, trial =1, start sec = 1126644713, CC stat = -5.655936e-05, theor sigma = 4.181445e-04
segment=41, trial =1, start sec = 1126644743, CC stat = 6.392874e-05, theor sigma = 4.092614e-04
segment=42, trial =1, start sec = 1126644773, CC stat = 1.122695e-05, theor sigma = 4.149807e-04
segment=43, trial =1, start sec = 1126644803, CC stat = -1.060125e-05, theor sigma = 4.130287e-04
segment=44, trial =1, start sec = 1126644833, CC stat = 2.608625e-04, theor sigma = 4.083948e-04
segment=45, trial =1, start sec = 1126644863, CC stat = 2.561162e-04, theor sigma = 4.105651e-04
segment=46, trial =1, start sec = 1126644893, CC stat = 5.447520e-05, theor sigma = 4.234874e-04
segment=47, trial =1, start sec = 1126644923, CC stat = -2.596327e-04, theor sigma = 4.356204e-04
segment=48, trial =1, start sec = 1126644953, CC stat = -1.167126e-04, theor sigma = 4.279144e-04
segment=49, trial =1, start sec = 1126644983, CC stat = 5.111158e-04, theor sigma = 4.216425e-04
segment=50, trial =1, start sec = 1126645013, CC stat = -5.790996e-04, theor sigma = 4.293588e-04
segment=51, trial =1, start sec = 1126645043, CC stat = -4.878459e-04, theor sigma = 4.467978e-04
segment=52, trial =1, start sec = 1126645073, CC stat = -6.029956e-05, theor sigma = 4.523662e-04
segment=53, trial =1, start sec = 1126645103, CC stat = 8.057724e-04, theor sigma = 4.427342e-04
